src/easystar.js

```diff
@@ -431,6 +449,23 @@ EasyStar.js = function() {
         return false;
     };
 
+    /**
+     * -1, -1 | 0, -1  | 1, -1
+     * -1,  0 | SOURCE | 1,  0
+     * -1,  1 | 0,  1  | 1,  1
+     */
+    var calculateDirection = function (diffX, diffY) {
+        if (diffX === 0, diffY === -1) return 'bottom'
+        else if (diffX === 1, diffY === -1) return 'bottom-left'
```

Comment:
I'm personally not fond of simple constants like that, as it just adds a lot of boilerplate without adding too much value. But if you like them I can include it.